client/my-sites/checkout/checkout-thank-you/google-apps-details.jsx

```diff
@@ -95,7 +102,11 @@ const GoogleAppsDetails = ( { purchases } ) => {
 					</p>
 				</div>
 			}
-			buttonText={ i18n.translate( 'Learn more about G Suite' ) }
+			buttonText={ i18n.translate( 'Learn more about %(productName)s', {
```

Comment:
In this particular case, I feel using the family name instead of the product name would be more appropriate.